client/client.go

```diff
@@ -159,7 +159,10 @@ type PrmInit struct {
 // SetDefaultSigner sets Client private signer to be used for the protocol
 // communication by default.
 //
-// Required for operations without custom signer parametrization (see corresponding Prm* docs).
+// Optional if you intend to sign every request separately, but required if
+// you'd like to use this signer for all operations implicitly Prm* docs).
```

Comment:
u either wanted to remove Prm mention or lost a part of it

Reply:
Fixed.